pkg/cloudprovider/plugins/internal/cloud_common.go

```diff
@@ -233,14 +219,13 @@ func (c *cloudCommon) ResetInventoryCache(accountNamespacedName *types.Namespace
 	return nil
 }
 
-// GetVpcInventory gets a map of vpcs applicable for the account.
-func (c *cloudCommon) GetVpcInventory(accountNamespacedName *types.NamespacedName) (map[string]*runtimev1alpha1.Vpc, error) {
+func (c *cloudCommon) GetCloudInventory(accountNamespacedName *types.NamespacedName) (*nephetypes.CloudInventory, error) {
```

Comment:
Add a comment for the function. 
The other functions have a description.

Reply:
done